plugin/src/main/java/net/codingarea/challenges/plugin/utils/item/DefaultItem.java

```diff
@@ -26,12 +28,24 @@ public static String getItemPrefix() {
 
 	@Nonnull
 	public static ItemBuilder navigateBack() {
-		return new SkullBuilder(ARROW_LEFT_UUID, "MHF_ArrowLeft").setName(Message.forName("navigate-back")).hideAttributes();
```

Comment:
is there a reason for using url streaming instead of player uuid?

Reply:
We should use the texture String directly instead of URLs or uuids.